src/app/static/src/app/components/projects/ShareProject.vue

```diff
@@ -119,6 +122,13 @@
         methods: {
             cloneProject: mapActionByName("projects", "cloneProject"),
             userExists: mapActionByName("projects", "userExists"),
+            enterEmails(email: EmailToShareWith, index: number){
```

Comment:
Expected behaviour: user enters email in first input > validation > new input created and is focused automatically > if user does not enter anything into the empty input and presses enter the focus moves to the ok button (if not disabled, otherwise just blurs the input) > if user presses enter again the ok button is triggered.

Reply:
After reviewing the accessibility ticket, it occurs to me that the first input should be automatically focused when the dialog is opened.

Reply:
Arguably for accessibility purposes (and to benefit screen readers etc) we should be following the default behaviour of HTML forms i.e. pressing enter on any input submits the form, and tab cycles through inputs + buttons.

Reply:
I think this is a special case because pressing enter submits the inputs for validation against the database and each other (ie, checks for duplicates, etc), and as such isn't as passive as simply tabbing. Jeff requested this functionality, but I agree tabbing to move through the inputs would work as well. We can present to Jeff as is and review in future if there is a better approach, taking into account accessibility concerns in a future ticket.